src/sentry/search/snuba/backend.py

```diff
@@ -340,15 +343,29 @@ def __run_joined_query_and_log_metric(
                     "snuba.search.group_attributes_joined.events_compared",
                     tags={"comparison": comparison},
                 )
-        except InvalidQueryForExecutor:
-            logging.info(
+
+            metrics.incr("snuba.search.group_attributes_joined.query", tags={"exception": "none"})
+        except InvalidQueryForExecutor as e:
+            logger.info(
                 "unsupported query received in GroupAttributesPostgresSnubaQueryExecutor",
                 exc_info=True,
             )
-        except Exception:
-            logging.warning(
+            metrics.incr(
+                "snuba.search.group_attributes_joined.query",
+                tags={
+                    "exception": f"{type(e).__module__}.{type(e).__qualname__}",
```

Comment:
Do you have an idea of how many tag values there could be here?

Reply:
There should probably just be a handful imo.